packages/vats/src/core/startWalletFactory.js

```diff
@@ -270,6 +272,7 @@ export const WALLET_FACTORY_MANIFEST = {
       namesByAddressAdmin: true,
       startUpgradable: true,
       startGovernedUpgradable: true,
+      economicCommitteeCreatorFacet: true,
```

Comment:
I was prompted by this to check the upgrade considerations in this PR. It feels like they could fill a small book. But I think the short version is: mostly harmless.